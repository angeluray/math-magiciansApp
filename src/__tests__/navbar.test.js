import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import MathNavbar from '../components/navbar';

describe('Navbar component testing using snapshots', () => {
  it('Test Navbar component renders', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <MathNavbar />
      </BrowserRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
