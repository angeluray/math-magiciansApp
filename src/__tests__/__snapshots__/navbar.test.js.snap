// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component testing using snapshots Test Navbar component renders 1`] = `
<nav
  className="navBar"
>
  <h1
    className="navTitle"
  >
    Math Magicians
  </h1>
  <ul>
    <li
      className="li1"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="li2"
    >
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li3"
    >
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
